tests/test_dvc.py

```diff
@@ -341,3 +338,106 @@ def test_warn_on_dvcyaml_output_overlap(tmp_dir, mocker, mocked_dvc_repo, dvcyam
         logger.warning.assert_called_with(msg)
     else:
         logger.warning.assert_not_called()
+
+
+def test_get_dvc_stage_template_empty(tmp_dir, mocked_dvc_repo):
+    live = Live()
+    template = get_dvc_stage_template(live)
+
+    assert template == {
+        "stages": {
+            "dvclive": {
+                "cmd": "<python my_code_file.py my_args>",
+                "deps": ["<my_code_file.py>"],
```

Comment:
q: why this mention has surrounding `<>` but not the one in cmd?

Reply:
cmd does have `<>`? did i misunderstand you?

btw I'm fine if you want to suggest something different for `cmd` and `deps`

Reply:
oh my bad, I didn't notice that and was wondering why my_code_file.py was not surrounded in cmd